### PR TITLE
Update message mapping config

### PR DESCRIPTION
**Pull request by qixiang**
This is for the changes in MBS:

https://pagure.io/fm-orchestrator/c/7555465365d410fa1bbd61c9406969905691278a?branch=v3
https://pagure.io/fm-orchestrator/c/84ffa8ed217e1f01520905ab367eca790f5e2524?branch=v3

### CODE REVIEW

**Comment by qixiang** [comment]
Hi @mikebonnet, could you help to review this and bump the version if PR is ok?

File: conf/config.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
         ],
         'topic': 'topic',
         'msg_id': 'headers.message-id',
-        'build_id': 'body.info.id',
```
Comment (mikebonnet) [inline conf/config.py:12]: Is there a good reason to remove the `build_id` and `build_name` fields? Seems like leaving them in the mapping shouldn't cause problems, and they may be useful in the future.

**Review by mikebonnet — COMMENTED** [review]
When do you expect to be merging the `v3` branch? Would you need a rebuild of the rpm right away, or would you want to delay it?

File: conf/config.py (hunk 1 above)
Comment (qixiang) [reply conf/config.py:12]: The parsed message from mbs-messaging-umb will be converted to a dict and passed to handlers in MBS in format of `func(**kwargs)`, `build_id` and `build_name` are not used in the handlers at all, so they were removed from handler args, the change of `repo_tag` to `tag_name` is for implementing MBS task router in a more easier way,  after the change, calling handlers with these args will result in unexpected keyword argument error.

**Comment by qixiang** [comment]
> When do you expect to be merging the `v3` branch? Would you need a rebuild of the rpm right away, or would you want to delay it?

We don't have an exact time at this moment, it should happen in a near future. Rebuilding the rpm with this change and update package in repo may break our current deployment, so we don't expect a package update now, only code change is sufficient. We can rebuild rpm later when we're ready for the v3 deployment.

File: conf/config.py (hunk 1 above)
Comment (mikebonnet) [reply conf/config.py:12]: I see, thanks for the explanation.

**Comment by mikebonnet** [comment]
Looks good! I'll merge this and bump the version. Thanks!

**Pull request merged**